client/src/app/pages/applications/components/application-list-expanded-area/application-list-expanded-area-extras.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { TableComposable, Thead, IRow, Tr, Th, Tbody, Td } from '@patternfly/react-table';
+
+import {
+  DescriptionList,
+  DescriptionListDescription,
+  DescriptionListGroup,
+  DescriptionListTerm,
+} from "@patternfly/react-core";
+
+import { Application, Task } from "@app/api/models";
+import { ApplicationTags } from "../application-tags";
+import { ApplicationAnalysisStatus } from "../../components/application-analysis-status";
+
+export interface IApplicationListExpandedAreaProps {
+  application: Application;
+  tasks: Task[];
+}
+
+export const ApplicationListExpandedAreaExtras: React.FC<
+  IApplicationListExpandedAreaProps
+> = ({ application, tasks }) => {
+  const { t } = useTranslation();
+
+  const rows: IRow[] = [];
+  tasks?.forEach((task) => {
+    rows.push(
+      <Tr key="Name">
+        <Td dataLabel="Addon">{task.addon}</Td>
+        <Td dataLabel="Status">
+          <ApplicationAnalysisStatus state={task.state? task.state : "No task"} />
+        </Td>
+      </Tr>
+    )
+  })
+
+  var tasksGroup;
+  if (tasks.length > 0) {
+    tasksGroup = (
+      <DescriptionListGroup>
+        <DescriptionListTerm>{t("terms.tasks")}</DescriptionListTerm>
+        <TableComposable aria-label="Simple table">
```

Comment:
We could wrap the table in a description list description so that the addons table takes up the entire width of the container. 
![image](https://user-images.githubusercontent.com/11218376/211357401-011d41f8-3b38-4676-965d-7ef480a99223.png)

Reply:
You may also want to use `variant="compact"` on the inner table, and maybe also `borders={false}`.